api/src/main/java/io/opentelemetry/trace/SemanticConventions.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace;
+
+/**
+ * Defines constants for all attribute names defined in the OpenTelemetry Semantic Conventions
+ * specifications.
+ *
+ * @see <a
+ *     href="https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md">Semantic
+ *     Conventions</a>
+ */
+public final class SemanticConventions {
```

Comment:
Since these all apply to attributes, could we rename the class to `SemanticAttributes`?

Reply:
I think it is a good suggestion.

Reply:
I like `SemanticAttributes` more as well.

Reply:
Changed